### PR TITLE
Add comments from `util.ts` to `util.js`

### PR DESCRIPTION
**Pull request by Saibamen**

### CODE REVIEW

**Comment by gaby** [comment]
👍🏻 Looks good!

**Review by gaby — APPROVED** [review]

**Comment by louislam** [comment]
The util.js is generated code. Cannot be modified manually.

**Comment by chakflying** [comment]
Maybe we should just add `"removeComments": false` to `tsconfig.json` instead.

File: src/util.ts [hunk 1]
```diff
@@ -50,24 +50,22 @@ export function debug(msg: any) {
 
 declare global { interface String { replaceAll(str: string, newStr: string): string; } }
 
+/**
+ * String.prototype.replaceAll() polyfill
+ * https://gomakethings.com/how-to-replace-a-section-of-a-string-with-another-one-with-vanilla-js/
+ * @author Chris Ferdinandi
+ * @license MIT
```
Comment (louislam) [inline src/util.ts:57]: It should not be moved to outside, as it is a general polyfill function, not only for replaceAll()

**Comment by Saibamen** [comment]
I changed `removeComments` to false

File: src/util.ts (hunk 1 above)
Comment (Saibamen) [reply src/util.ts]: But without this, we are missing a tooltip for this function

Comment (louislam) [reply src/util.ts]: Can add another one if needed.

Definition from the Internet:

> A polyfill is a piece of code used to provide modern functionality on older browsers that do not natively support it

Comment (Saibamen) [reply src/util.ts]: I moved it to the place where it was before. Other comments can be added by you or in different PRs

**Pull request merged**